### ISSUE TITLE
Query about reproducing the given postraining checkpoint

### ISSUE THREAD

**Issue by Char-lotte-Xia** [Robbyant/lingbot-va#52]
Hi, I appreciate your work very much, but I have some problems in reproducing the posttraining results as well as the results in the essay.

1. I didn't change any setting of robotwin2.0 in the repo (including the seed), and directly tested your rank-0 tasks on the downloaded `lingbot-va-posttrain-robotwin` checkpoint, but half of the tasks have dropped by more than 10%.  such as `scan_object`, `stack_blocks_three`, `place_shoe`. But the other half works well. I hope you can reassure that this process is the same with your evaluation.

2. I tried to posttrain `lingbot-va-base` on `robotwin-clean-and-aug-lerobot`, both downloaded from your repo, and use the default training params from your repo, but the result is worse than your checkpoint. 
For instance, in your essay, you mentioned the training lr = 0.1, but didn't mention the posttraining lr. Also, in previous issues, you mentioned that the batch size can be set to 8. I'm curious about your posttraining dataset and config.
```python
from easydict import EasyDict
from .va_robotwin_cfg import va_robotwin_cfg
import os

va_robotwin_train_cfg = EasyDict(__name__='Config: VA robotwin train')
va_robotwin_train_cfg.update(va_robotwin_cfg)

# va_robotwin_train_cfg.resume_from = '/robby/share/Robotics/lilin1/code/Wan_VA_Release/train_out/checkpoints/checkpoint_step_10'
va_robotwin_train_cfg.wan22_pretrained_model_name_or_path = "xxxxxxx/checkpoint/lingbot-va-base"

va_robotwin_train_cfg.dataset_path = 'xxxxxx/data/robotwin-clean-and-aug-lerobot'
va_robotwin_train_cfg.empty_emb_path = os.path.join(va_robotwin_train_cfg.dataset_path, 'empty_emb.pt')
va_robotwin_train_cfg.enable_wandb = True
va_robotwin_train_cfg.load_worker = 16
va_robotwin_train_cfg.save_interval = 250
va_robotwin_train_cfg.gc_interval = 50
va_robotwin_train_cfg.cfg_prob = 0.1

# Training parameters
va_robotwin_train_cfg.learning_rate = 1e-5
va_robotwin_train_cfg.beta1 = 0.9
va_robotwin_train_cfg.beta2 = 0.95
va_robotwin_train_cfg.weight_decay = 0.1
va_robotwin_train_cfg.warmup_steps = 10
va_robotwin_train_cfg.batch_size = 1 
va_robotwin_train_cfg.gradient_accumulation_steps = 1
va_robotwin_train_cfg.num_steps = 40000
```

3. I found the evaluation procedure on robotwin rather time consuming, with evaluating one trajectory in one task costing more than 20 minutes. I use 8 4090, and launched the program with:

```python
save_root="results_post_all/"
task_group_id=0
bash evaluation/robotwin/launch_client_multigpus.sh ${save_root} ${task_group_id}
```
I wonder whether it is normal, and whether it aligns with your evaluation in the essay.

Hope to get your reply! Have a nice day!

**Comment by Char-lotte-Xia** [comment]
> I've successfully reproduced the result in the essay by changing sapien version to `3.0.0b1`(this changed the sr rate from 0.18 to 0.9 for handover block), and switching the version of robotwin by `cd RoboTwin && git checkout 2eeec322`. `attn-mode` is also important. 

 _Originally posted by @Char-lotte-Xia in [#27](https://github.com/Robbyant/lingbot-va/issues/27#issuecomment-4146806301)_


I've successfully reproduced the result.
